assets/css/evaluation.less

```diff
@@ -0,0 +1,76 @@
+body {
+    position: relative;
+    margin: 0;
+    padding-bottom: 6rem;
+    min-height: 100%;
+}
+
+
+#tbHeader {
+    background-color: #04b8f9;
+}
+
+#Logout {
+    float:right;
+    margin-right: 3rem;
+    border: none;
+    background-color: white;
+    font-size: 140%;
+}
+
+.borderSwitch{
+    border-bottom: solid thick #cc935b;
+    text-align: center;
+}
+
+.pgSwitch{
+    text-decoration: none;
+    color:black;
+    :visited color: black;
```

Comment:
I'm still not sure what this particular line (L29) is trying to achieve. Can you explain, please?